### ISSUE TITLE
ComponentGroupRef are not inserted in any feature

### ISSUE THREAD

**Issue by Leogiciel** [tom-englert/Wax#23]
Documentation says : 
Wax will add a ComponentGroupRef node for all component groups it creates to the first feature it finds in your project. If you have just one feature defined, this will be fine; if you have more than one feature in your setup project, copy or move the entries as desired.
```
  <Product ...>
    <Feature Id="ProductFeature" Title="ACME.Setup" Level="1">
      <ComponentGroupRef Id="ProductComponents" />
      <ComponentGroupRef Id="de_files" />
      <ComponentGroupRef Id="fr_files" />
    </Feature>
  </Product>
```
But no componentGroupRef is generated in my Wix project.

**Comment by tom-englert** [comment]
It works fine for me - can you debug this so see what happens? Add a break point at the method ForceFeatureRef()

**Comment by Leogiciel** [comment]
The method adds effectively a componentgrouref to first found feature object, but nothing changes in the .wxs file...

**Comment by Leogiciel** [comment]
I'm trying to understand and fix this...

**Comment by Leogiciel** [comment]
Freshly created "bin_files" ComponentGroup is not found in my whole project as a ComponentGroupRef : 

![capture2](https://cloud.githubusercontent.com/assets/1555868/21767592/5dcfa968-d675-11e6-81ec-2eee4b22f247.PNG)

My only feature : 
![capture2](https://cloud.githubusercontent.com/assets/1555868/21767659/ab494654-d675-11e6-9fae-af7960e3a126.PNG)

**Comment by tom-englert** [comment]
Maybe just the "Save" was missing? I think you added this in your last PR, right?

**Comment by Leogiciel** [comment]
Exactly, SourceFile.Save did the trick. 

Fixed by commit [fe5275b](https://github.com/tom-englert/Wax/commit/fe5275b81a3f674688cc36c1332ab41f094ef3d4) #30
